### ISSUE TITLE
Add: namespace and use keywords

### ISSUE THREAD

**Issue by eosrei** [chapter-three/oophp-training#3]
If this training is going to be re-used, one of the lessons should cover the `namespace` and the `use` keywords since both are relevant to Drupal 8.

http://php.net/manual/en/language.namespaces.php
http://php.net/manual/en/language.namespaces.importing.php

``` php
namespace Drupal\demo\Plugin\Block;

use Drupal\block\BlockBase;
use Drupal\Core\Session\AccountInterface;
```

src: http://www.sitepoint.com/building-drupal-8-module-blocks-forms/

**Comment by kscheirer** [comment]
Looks like this was added in 7.md. However, that section could use some excercises. May be hard to come up with something since we're not doing these exercises in a Drupal context.

**Comment by eosrei** [comment]
Ah ha! Yes it was mostly fixed in https://github.com/chapter-three/oophp-training/commit/97230c27b35eedd9ca493e429ca655e2ab64cd5b

You can `git commit --amend` #11 to close this issue IMO.

**Comment by mrf** [comment]
Merged #11 thanks for the addition!
